flatgfa/src/packedseq.rs

```diff
@@ -204,20 +293,33 @@ pub fn create_slice(vec: &PackedVec, s: std::ops::Range<usize>) -> PackedSlice<'
 
 /// Returns a vector containing the base pairs referenced by `slice`
 pub fn get_slice_seq(slice: PackedSlice<'_>) -> Vec<Nucleotide> {
-    slice.vec_ref.get_range(slice.span)
+    slice.vec_ref.as_ref().get_range(slice.span)
+}
+
+pub fn total_bytes(num_elems: usize) -> usize {
```

Comment:
Not 100% sure about this one, but perhaps this API could be a tiny bit friendlier if it took a `PackedSeqStore` or `PackedSeqView` as an argument. It would of course just immediately use `.len()` to get the size, but it would make it a little clearer to callers that they're supposed to use this to get the file size for one of these data structures.